plugins/modules/homebrew_services.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Kit Ham <kitizz.devside@gmail.com>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+---
+module: homebrew_services
+author:
+    - "Kit Ham (@kitizz)"
+requirements:
+    - homebrew must already be installed on the target system
+short_description: Services manager for Homebrew
+description:
+    - Manages daemons and services via Homebrew.
+extends_documentation_fragment:
+    - community.general.attributes
+attributes:
+    check_mode:
+        support: full
+    diff_mode:
+        support: none
+options:
+    name:
+        description:
+            - An installed homebrew package whose service is to be updated.
+        aliases: [ 'formula', 'package', 'pkg' ]
```

Comment:
In general adding aliases should be avoided in not necessary. I can see why `formula` makes sense, but are `package` and `pkg` really needed?

Reply:
Mainly matching the aliases in the other homebrew modules for consistency. But if these are unorthodox, I'm happy to nix them.